plugins/pulp_rpm/plugins/importers/yum/repomd/updateinfo.py

```diff
@@ -106,6 +106,13 @@ def _parse_collection(element):
     ret = {
         'packages': map(_parse_package, element.findall('package')),
     }
+
+    # a module element indicates this erratum being part of a module
+    modulemds = element.findall('module')
```

Comment:
Why modulemds are in plural? There could be 0 or 1 modulemd in a collection.

Reply:
to match the return type of the `element.findall()` method